superset/connectors/base/models.py

```diff
@@ -86,7 +86,7 @@ def uid(self):
 
     @property
     def column_names(self):
-        return sorted([c.column_name for c in self.columns])
+        return sorted([c.column_name or '' for c in self.columns])
```

Comment:
@graceguo-supercat the issue here is we would be replacing `None` with an empty string in the sorted list. Maybe something like this would be preferred, 
```
sorted([c.colum_name for c in self.columns], key=lambda x: x or '')
```